package.json

```
@@ -4,7 +4,7 @@
   "version": "1.1.1",
   "description": "💊 Valtio makes proxy-state simple for React and Vanilla",
   "main": "./index.js",
-  "module": "./esm/index.mjs",
+  "module": "./esm/index.js",
   "types": "index.d.ts",
```

```suggestion
  "types": "./index.d.ts",
```
maybe this looks more consistent?